docs/deploying.md

```diff
@@ -89,6 +89,8 @@ docker run -d -p 5000:5000 \
 	registry:2
 ```
 
+If the certificate issuer supplies you with an 'intermediate' certificate, such as Gandi, you need to combine your certificate with the intermediates to form a 'certificate bundle'. You can do this using the cat command: ```cat server.crt GandiStandardSSLCA2.pem > server.with-intermediate.crt```. You can then configure the registry to use your certificate bundle with the ```REGISTRY_HTTP_TLS_CERTIFICATE``` environment variable.
```

Comment:
Hmm, I'm not sure about linking to the wikipedia page, I doubt users are going to find themselves in a situation where they're not sure if they have an intermediate certificate.